Python/Location-based-on-IP/README.md

```diff
@@ -0,0 +1,20 @@
+## Geo-location based on IP address
+- This script can be used to obtain the latitude and longitude from where a particular IP address originates
+- The API from https://ip-api.com/ has been used for the same
+
+## Usage
+- Requirements 
```

Comment:
please specify commands to install requirements.

Reply:
Done